CombinatorialGames/Game/IGame.lean

```diff
@@ -1398,8 +1400,9 @@ private theorem invRec' {x : IGame} (hx : 0 < x)
   on_goal 5 => apply left₂
   on_goal 9 => apply right₁
   on_goal 13 => apply right₂
-  any_goals simpa [inv_eq hx, InvTy.val]
+  any_goals simp [inv_eq hx, InvTy.val]
   all_goals first |
+    exact by assumption |
```

Comment:
I'm confused by what the motivation for this change was.

Reply:
We had a `set_option` disabling the unnecessary simpa linter, rewriting the code like this means we don't need it anymore.